Fixed #24638 -- Added support for SQL comments (#18258)
django/django: docs/ref/models/querysets.txt

  filtering. As such, it should generally be called from the ``Manager`` or
  from a fresh ``QuerySet`` instance.

``comment()``
~~~~~~~~~~~~~

.. versionadded:: 5.2

.. method:: comment(message)

Adds an SQL comment to be inserted into the resultant SQL statement, after

> ``SELECT``/``UPDATE``, and if applicable, ``DISTINCT``.. For example::

Reviewer: Is it not possible to make it work for `INSERT`, at least when using `Model.objects.comment(...).create(...)` ?

    Entry.objects.comment("blog/views.py:50").all()

… translates (roughly) into the following SQL::

    SELECT /* blog/views.py:50 */ blog_entry.* FROM blog_entry

> This is useful for adding information to the SQL statement which when logged
> can be used to correlate slow and problematic queries to the generating source
> code.

Reviewer: ```suggestion
This is useful for adding logging information to the generated SQL for
correlating problematic queries with the responsible Python code.
```

Comments can be chained and will be added to the SQL statement as separate
entries, for example::

    Entry.objects.comment("blog/views.py:50").comment("python=3.11").update(
        headline="This is new"
    )

… translates into::

>     UPDATE blog_entry.* /* blog/views.py:75 */ /* python=3.11 */ SET "headline" = 'This is new'

Reviewer: Needs updating
```suggestion
    UPDATE /* blog/views.py:75 */ /* python=3.11 */ blog_entry SET "headline" = 'This is new'
```
